integration-tests/src/test/java/org/zowe/apiml/gatewayservice/ZosmfSsoIntegrationTest.java

```diff
@@ -126,9 +126,7 @@ public void doZosmfCallWithInvalidToken() {
         .when()
             .get(String.format("%s://%s:%d%s%s", scheme, host, port, BASE_PATH, ZOSMF_ENDPOINT))
         .then()
-            .statusCode(is(SC_UNAUTHORIZED))
```

Comment:
What is the current message if not ZWAG102E?

Reply:
It is same like before issue (with JWTValidationFilter), it is stopped by ServiceAuthenticationFilter. There cannot be a concrete message.

Reply:
I see. Thank you